packages/ibm-products/src/components/ActionBar/ActionBar.tsx

```diff
@@ -30,6 +38,59 @@ const defaults = {
   actions: Object.freeze([]),
 };
 
+interface Action extends ButtonProps {
+  id?: string;
+  key: string;
+  iconDescription: string;
+  label: string;
+  onClick: () => void;
+  renderIcon: CarbonIconType;
```

Comment:
Hey,
Looks like `onClick` is an optional prop.
Also why we have included `id` property which was not part of the proptypes

Reply:
updated onclick to optional! and added is to proptypes since it's being included here: https://github.com/carbon-design-system/ibm-products/pull/5733/files/03f855ba4effe7dd99cd9591e544a70f40ae76d7#diff-b3bc1c13c1dde53425d434e6bf787214e52de32d4302f2d95339c0043eb11ca0R146